diff --git a/node_modules/expo/ios/AppDelegates/EXReactRootViewFactory.h b/node_modules/expo/ios/AppDelegates/EXReactRootViewFactory.h
index 2daec5d..dca5ed2 100644
--- a/node_modules/expo/ios/AppDelegates/EXReactRootViewFactory.h
+++ b/node_modules/expo/ios/AppDelegates/EXReactRootViewFactory.h
@@ -6,6 +6,7 @@
 #import <Expo/RCTAppDelegateUmbrella.h>
 
 NS_ASSUME_NONNULL_BEGIN
+@class RCTDevMenuConfiguration;
 
 @class EXReactDelegate;
 
diff --git a/node_modules/expo/ios/AppDelegates/ExpoReactNativeFactory.swift b/node_modules/expo/ios/AppDelegates/ExpoReactNativeFactory.swift
index 925e0f9..371bd82 100644
--- a/node_modules/expo/ios/AppDelegates/ExpoReactNativeFactory.swift
+++ b/node_modules/expo/ios/AppDelegates/ExpoReactNativeFactory.swift
@@ -117,8 +117,7 @@ public class ExpoReactNativeFactory: ExpoReactNativeFactoryObjC, ExpoReactNative
       rootView = factory.superView(
         withModuleName: moduleName ?? defaultModuleName,
         initialProperties: initialProps,
-        launchOptions: launchOptions ?? [:],
-        devMenuConfiguration: self.devMenuConfiguration
+        launchOptions: launchOptions ?? [:]
       )
 #else
       rootView = factory.superView(
@@ -132,8 +131,7 @@ public class ExpoReactNativeFactory: ExpoReactNativeFactoryObjC, ExpoReactNative
       rootView = rootViewFactory.view(
         withModuleName: moduleName ?? defaultModuleName,
         initialProperties: initialProps,
-        launchOptions: launchOptions,
-        devMenuConfiguration: self.devMenuConfiguration
+        launchOptions: launchOptions
       )
 #else
       rootView = rootViewFactory.view(
